oak/server/rust/oak_runtime/tests/integration_test.rs

```diff
@@ -109,3 +121,24 @@ fn test_metrics_gives_the_correct_number_of_nodes() {
 
     runtime.stop_runtime();
 }
+
+#[test]
+#[ignore]
+/// Gets Roughtime from the live default servers with the default settings.
+///
+/// This requires an internet connection and at least 3 of the default servers to be operational and
+/// accessible.
```

Comment:
Does it also require the local machine's clock to be vaguely accurate?  (Mention in the comment if so).

Reply:
Done.